src/app/fhir-constants.ts

```diff
@@ -0,0 +1,225 @@
+/**
+ * FHIR types.
+ */
+
+/**
+ * Research study status, from
+ * https://www.hl7.org/fhir/valueset-research-study-status.html
+ */
+export enum ResearchStudyStatus {
+  /**
+   * Active
+   * Study is opened for accrual.
+   */
+  ACTIVE = 'active',
```

Comment:
Just a personal preference, I think there should be a new line to separate each enum value and the comment beneath it.

Reply:
We can change this in a future update, this is basically just the end result of running a regexp over a copy-paste of the table from the FHIR spec.